src/internals/ChartInternal.js

```diff
@@ -946,7 +946,8 @@ export default class ChartInternal {
 			y = isRotated ? 0 : $$.height2;
 		} else if (target === "arc") {
 			x = $$.arcWidth / 2;
-			y = $$.arcHeight / 2;
+			// to prevent wrong display of min and max label in case when legend is not at the bottom
+			y = $$.arcHeight / 2 - (($$.hasType("gauge") && this.config.legend_position === "bottom") ? 0 : 6);
```

Comment:
This affects Arc's expands area display. Updating a general value should be considered with more cautions.
![image](https://user-images.githubusercontent.com/2178435/72118392-53a94500-3394-11ea-918a-df9fcc42dbe4.png)